grouper/models/group_edge.py

```diff
@@ -115,10 +116,12 @@ def apply_changes(self, changes):
                     # Check for and remove pending expiration notification.
                     cancel_expiration(self.session, group_name, member_name)
                 if value:
-                    expiration = datetime.strptime(value, "%m/%d/%Y")
+                    expiration = _make_date_obj(value)
+                    assert isinstance(expiration, datetime)
```

Comment:
I don't think this assert is necessary; if `_make_date_obj` promises to always return a datetime we don't need to double check it.